conn/physic/doc.go

```diff
@@ -6,4 +6,19 @@
 // units.
 //
 // This includes temperature, humidity, pressure, tension, current, etc.
+//
+//
+// SI units
+//
+// The supported S.I. units is a subset of the official ones.
+//   T		tera		10¹²		1000000000000
+//   G		giga		10⁹			1000000000
```

Comment:
Use space characters instead of tabs to ensure alignment is 100% consistent.

Reply:
I couldn't quite get it right with just spaces in both the editor and godoc, I had to use spaces and tabs. I'm sure its to fragile to survive.